src/Charts/PopoverChartSelector.js

```diff
@@ -92,32 +92,27 @@ export function PopoverChartSelector(evt, placement = 'bottom', alignment = 'sta
             position.left = bbox.left - popover.clientWidth - padding;
         }
 
-        const popoverRight = popover.clientWidth + position.left + padding;
```

Comment:
if you didn't do any changes with this popover position stuff please change all this back